src/main/java/uk/gov/justice/digital/hmpps/cmd/api/dto/NotificationDto.kt

```diff
@@ -17,8 +17,8 @@ data class NotificationDto @JsonCreator constructor(
         @JsonProperty("lastModified")
         val lastModified: LocalDateTime,
         @ApiModelProperty(required = true, value = "Whether the notification has been acknowledged", position = 3, example = "true")
-        @JsonProperty("read")
-        val read: Boolean
+        @JsonProperty("acknowledged")
+        val acknowledged: Boolean
```

Comment:
Would `seen` be a simpler word here?

Reply:
its 'has been seen on the UI, emailed out, sms'd out or ignored' I'm up for a shorter word though

Reply:
`seen` is not much different to `read`